### PR TITLE
feat: add weighted ecdsa validator plugin

### PR DESCRIPTION
**Pull request by adnpark**

### CODE REVIEW

File: plugins/weightedECDSA/index.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import type { KernelValidator } from "@zerodev/sdk/types"
+import {
+    createWeightedECDSAValidator,
+    updateConfig
+} from "./toWeigthedECDSAValidatorPlugin.js"
+
+export { createWeightedECDSAValidator, updateConfig, type KernelValidator }
```
Comment (derekchiang) [inline plugins/weightedECDSA/index.ts:7]: Why are you re-exporting the `KernelValidator` type?

Comment (adnpark) [reply plugins/weightedECDSA/index.ts:7]: I've copied the existing ecdsa plugin and it re-exports `KernelValidator` type. Not sure but I've just followed the ecdsa plugin code.

Comment (adnpark) [reply plugins/weightedECDSA/index.ts:7]: If we can remove it, then I'll delete it in ecdsa plugin too

Comment (derekchiang) [reply plugins/weightedECDSA/index.ts:7]: Oh hmm @SahilVasava why are we re-exporting it?

File: plugins/weightedECDSA/toWeigthedECDSAValidatorPlugin.ts [hunk 2]
```diff
@@ -0,0 +1,201 @@
+import { KERNEL_ADDRESSES } from "@zerodev/sdk"
```
Comment (derekchiang) [inline plugins/weightedECDSA/toWeigthedECDSAValidatorPlugin.ts:1]: Your file name is wrong btw.  `Weighted`

**Pull request closed without merging**